pyArango/tests/tests.py

```diff
@@ -852,6 +852,14 @@ def test_users_update(self) :
 
         Connection(arangoURL=ARANGODB_URL, username="pyArangoTest_tesla", password="newpass")
 
+    def test_foxx(self):
```

Comment:
hm, shouldn't this be rather something other than fetching a html file? We meanwhile did a good job in migrating away most of aardvarks routes to native arangodb routes ;-) however there still seems to be i.e.: 
`POST /_db/_system/_admin/aardvark/graph-examples/create/mps_graph`
which probably will remain available and do something where you can revalidate that it worked - plus a parseable json reply. 

`/_db/_system/` should come from the selected db this foxx is working on. `_admin/aardvark` is the mountPoint, `create` the actual route, `mps_graph` a URL parameter.
if you put in a wrong graph, it throws an error, which you should convert to a python exception (imho?) 

I'd rather see the use of it a bit like that: you derive the foxx base class, implement the routes to query. You'll then create an instance of it by specifying its mountpoint. 
In optimal, the client stub could be generated from a swagger json, but by hand should be ok. So here: 
```
class aardvarkTest(_BaseFoxx):
      def __init__(self, db, mountPoint):....
      def graph_examples_create(self):
...
```
and then the instance: 
```
adminAardvark = db.getFoxx(aardvarkTest, "_admin/aardvark")
```

Reply:
Hi @dothebart ,
Can you direct me to the page which has the aardvarks routes?
So, I can add some examples.